app/views/account/users/edit.html.slim

```diff
@@ -0,0 +1,4 @@
+h1.col-md-6.col-md-offset-3 Edit User
+.row
+  .col-md-3.col-md-offset-3
+    = render 'account/users/form'
```

Comment:
try to replace this with `= render 'form'` to keep consistent with 
`app/views/account/users/new.html.slim`

also, good practice to render forms like this:

```
# app/views/account/users/new.html.slim
= simple_form_for [:account, @user] do |form|
  = render 'form', f: form

# app/views/account/users/_form.html.slim
= f.input :first_name, autofocus: true
= f.input :last_name
= f.input :email
= f.input :role , collection: [User::ADMIN_ROLE,User::SALE_ROLE], include_blank: false
```